docs/workflows/common-workflow-steps/custom-script-from-git.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Custom Script from Git
+description: Use Script steps for additional functionalities in your builds.
+tags: [custom scripts, build, test, workflow, step]
```

Comment:
The tags "test" do not suit this component; "workflow" and "step" are too general for this component.
More relevant tags to add would be "git" and "git scripts" to better reflect the feature’s focus on running scripts from Git repositories.